frontend/wallet/src/render/views/onboarding/Onboarding.re

```diff
@@ -38,37 +38,38 @@ let make = () => {
     | CustomSetupB =>
       <CustomSetupB
         prevStep={_ => setOnboardingStep(_ => CustomSetupA)}
-        runNode={_ => setOnboardingStep(_ => RunNode)}
+        runNode={_ => setOnboardingStep(_ => RunNode(false))}
         nextStep={_ => setOnboardingStep(_ => CustomSetupC)}
       />
     | CustomSetupC =>
       <CustomSetupC
         prevStep={_ => setOnboardingStep(_ => CustomSetupB)}
```

Comment:
Does anyone know what the deal with CustomSetupB is? @michellewong793 
It's never the nextStep but here it's the prevStep

Reply:
We decided to move this out if I recall our conversation with Chris.